.github/workflows/ci.yml

```diff
@@ -40,3 +40,16 @@ jobs:
         with:
           command: test
           args: --workspace ${{ matrix.flags }}
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+            toolchain: nightly
```

Comment:
can we use the stable one instead? that way people don't need the nightly one on their machines to double-check for upcoming clippy lints